### PR TITLE
Add install.sh

### LINKED ISSUE #97: please add install guide  Linux binary

### PR DESCRIPTION
**Pull request by 178inaba**
I added an install script using [GoDownloader](https://github.com/goreleaser/godownloader).
And I added the installation method to README referring to [golangci-lint install page](https://golangci-lint.run/usage/install/#other-ci).

GoDownloader does not support binary release files, so it has been fix to release tar.gz files.
The latest release is a binary file release, so install.sh will not work.
A new release will be required when merging this pull request.

Incidentally, the combination of darwin and 386 is ignored because there is no corresponding machine.
This is also done in golangci-lint.
https://github.com/golangci/golangci-lint/blob/1844a75eec6cad489133c9d354ddc2e6717f6cc4/.goreleaser.yml#L30-L32

Fixes #97.

### CODE REVIEW

**Comment by xiantang** [comment]
let me check this...

**Comment by 178inaba** [comment]
@xiantang 
OK!
I checked with the following command.

```console
$ cat install.sh | sh -s -- -b $(go env GOPATH)/bin v1.12.2
$ cat install.sh | sh -s v1.12.2
```

`install.sh` was generated with the following command.

```console
$ godownloader --repo=cosmtrek/air --output=./install.sh ./.goreleaser.yml
$ chmod a+x ./install.sh
```

File: README.md [hunk 1]
```diff
@@ -38,7 +38,15 @@ go get -u github.com/cosmtrek/air
 
 ### macOS, Linux, Windows
 
-The binary is published on the [release page](https://github.com/cosmtrek/air/releases).
+```bash
+# binary will be $(go env GOPATH)/bin/air
+curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
+
+# or install it into ./bin/
+curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s
```
Comment (xiantang) [inline README.md:46]: if user not have go environment what

File: .goreleaser.yml [hunk 2]
```diff
@@ -3,5 +3,6 @@ builds:
       - linux
       - windows
       - darwin
-archives:
-  - format: binary
+    ignore:
```
[the remark this answers is not in the document]
Comment (178inaba) [reply .goreleaser.yml:6]: Fixed to upload binary files by specifying both `tar.gz` and `binary` in `archives`.

File: README.md (hunk 1 above)
Comment (178inaba) [reply README.md:46]: Yes.
The install script works even if Go runtime is not installed.

**Comment by xiantang** [comment]
LGTM

**Pull request merged**